deepchem/molnet/__init__.py

```diff
@@ -14,7 +14,7 @@
 from deepchem.molnet.load_function.lipo_datasets import load_lipo
 from deepchem.molnet.load_function.muv_datasets import load_muv
 from deepchem.molnet.load_function.nci_datasets import load_nci
-from deepchem.molnet.load_function.pcba_datasets import load_pcba
```

Comment:
For backwards compatibility, could you keep the old `load_pcba` import as well?

Reply:
Sure. Somehow the rebase took some of your old commits. I don't think it will cause any issues, but do not know how to avoid it.